modules/adding-a-custom-logo.adoc

```diff
@@ -11,7 +11,7 @@ You can create custom branding by adding a custom logo or custom product name. Y
 .Prerequisites
 
 * You must have administrator privileges.
-* Create a file of the logo that you want to use. The logo can be a file in any common image format, including GIF, JPG, PNG, or SVG, and is constrained to a `max-height` of `60px`.
+* Create a file of the logo that you want to use. The logo can be a file in any common image format, including GIF, JPG, PNG, or SVG, and is constrained to a `max-width` of `200px` and a max-height` of `68px`. Image size must not exceed `1 MB` due to constraints on the `ConfigMap` size.
```

Comment:
Consider using either a [general reference](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#general-references) or use an [object reference](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#object-references) for object names.

Further, about the size values in backticks: if  `max-width` and  `max-height` are parameters/specs, it seems valid to have these and their respective values in backticks; not sure if we need backticks for the 1 MB size limit, though.

```suggestion
* Create a file of the logo that you want to use. The logo can be a file in any common image format, including GIF, JPG, PNG, or SVG, and is constrained to a `max-width` of `200px` and a `max-height` of `68px`. Image size must not exceed 1 MB due to constraints on the `ConfigMap` object size.
```